src/processor/Package.ts

```diff
@@ -228,4 +230,44 @@ export class Package {
       pullAt(sd.rules, rulesToRemove);
     });
   }
+
+  private suppressUrlAssignmentOnExtensions(): void {
+    // Loop over all profiles and extensions, removing assignment rules on inline extensions
+    [...this.profiles, ...this.extensions].forEach(sd => {
+      const rulesToRemove: number[] = [];
+      sd.rules.forEach(rule => {
+        if (rule instanceof ExportableContainsRule && rule.path.endsWith('extension')) {
+          rule.items.forEach(item => {
+            const assignmentRuleIdx = sd.rules.findIndex(
+              other =>
+                other.path === `${rule.path}[${item.name}].url` &&
+                other instanceof ExportableFixedValueRule &&
```

Comment:
I can't explain why, but I feel like the `instanceof` should be the first thing to check.  It feels like maybe it is less expensive?  Or it more quickly sets the context for people reading the code?  I don't know.  You can ignore this if you want.